spec/unit/comments_spec.rb

```diff
@@ -17,15 +17,12 @@
     describe ".find_for_resource_in_namespace" do
       let(:post) { Post.create!(title: "Hello World") }
       let(:namespace_name) { "admin" }
-
-      before do
-        @comment = ActiveAdmin::Comment.create! resource: post,
-                                                body: "A Comment",
-                                                namespace: namespace_name
-      end
+      let!(:comment) { ActiveAdmin::Comment.create! resource: post,
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Block body expression is on the same line as the block start.